.changeset/moody-pants-own.md

```diff
@@ -1,23 +1,24 @@
 ---
-"@remix-run/dev": minor
+"@remix-run/dev": patch
 ---
 
-feat: remix optional segments
+Convention for Remix optional segments
 
-Allows for the creation of optional route segments by using parenthesis. For example:
-Creating the following file routes in remix `/($lang)/about`
-this will match the following routes
+Note that this only set ups the convention conversion for React Router.
+Optional segments won't be available in Remix until Remix is built on top of React Router v6.5
+
+Converts segments surrounded by parenthesis into optional segments for React Router.
+For example `/($lang)/about` will be converted to `/:lang?/about` in React Router.
+
+This means `/($lang/about)` would match:
```

Comment:
@pcattori do you mean here `/($lang)/about`?

Reply:
Yes, this got fixed in #4825